src/java/us/kbase/mobu/tester/ConfigLoader.java

```diff
@@ -47,9 +52,13 @@ public ConfigLoader(Properties props, boolean testMode,
         String tokenString = props.getProperty(testPrefix + "token");
         if (tokenString != null && tokenString.trim().isEmpty()) {
             tokenString = null;
+        }
```

Comment:
adding in some extra messages for help in debugging auth problems (useful when running tests via GitHub actions)

Reply:
This seems to reverse the precedence of where the sdk looks for the token, is that right? That being said, IMO the precedence should be test file > env.

Reply:
No -- if the token string is not in the test file, the environment will be checked. If KBASE_TEST_TOKEN is non-null and not empty, it will be used.